test/transcoding/relationals_select.ll

```diff
@@ -0,0 +1,130 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: spirv-val %t.spv
```

Comment:
Suggest to remove spirv-val for now.
It errors with
error: line 74: All OpVariable instructions in a function must be the first instructions in the first block.
  %arg1 = OpVariable %_ptr_Function_v4float Function

and it relates to https://github.com/KhronosGroup/SPIRV-LLVM-Translator/issues/1883